src/runtime_src/core/pcie/driver/linux/xocl/devices.h

```diff
@@ -1690,7 +1690,7 @@ struct xocl_subdev_map {
 #define MFG_RES_U50							\
 	((struct xocl_subdev_info []) {					\
 	 	XOCL_DEVINFO_FLASH_MFG_U50,				\
-	 	XOCL_DEVINFO_XMC_MFG_U50,				\
+		/* XOCL_DEVINFO_XMC_U50, */				\
```

Comment:
Can we put a big comment here as to why this is done so? It's better to have comments when we put any workaround in our code since they are not something logically reasonable.

Reply:
>add the following comments into the code
/* Temporary disable XMC for U50 Golden, see CR-1052500 for more info */

> also add comments into the CR:
For pu2, temporary disable CMC access for golden.
Add this as workaround and should be fixed later. https://confluence.xilinx.com/display/XSW/XRT+workaround+list

> also add CR links into the confluence page. So we are able to chase this workaround from code, or from confluence page or from CR itself.